documentation/asciidoc/computers/raspberry-pi/frequency-management.adoc

```diff
@@ -78,9 +86,9 @@ Temperature decreases use the same mapping with a 5°C **hysteresis**; fan speed
 
 At boot the fan is turned on, and the tachometer input is checked to see if the fan is spinning. If it is, then the `cooling_fan` device tree overlay is enabled. This overlay is in `bcm2712-rpi-5-b.dtb` by default, but with `status=disabled`.
 
-==== Fan connector pinout
+==== Raspberry Pi 5 fan connector pinout
```

Comment:
One more nit 😁 Thoughts on this header? It's at the same level as the other "fan" headers (I didn't want to make it too small). Normally I don't think I would separate this information out using a header, but in this case, the connector info is linked in the introduction.

Reply:
Seems like a reasonable compromise. Ideally we'd nest this within the Pi 5 section, but... that might go too deep. This seems to be the best place for this right now.